code/modules/reagents/chemistry/reagents/drinks_reagents.dm

```diff
@@ -173,32 +173,36 @@
 	id = "banana"
 	description = "The raw essence of a banana."
 	color = "#F6F834"
+	process_flags = ORGANIC | SYNTHETIC
 	drink_icon = "banana"
 	drink_name = "Glass of banana juice"
 	drink_desc = "The raw essence of a banana. HONK"
 	taste_description = "banana juice"
 
 /datum/reagent/consumable/drink/banana/on_mob_life(mob/living/M)
 	var/update_flags = STATUS_UPDATE_NONE
+	var/is_robot = ((process_flags & SYNTHETIC) > 0)
 	if(HAS_TRAIT(M, TRAIT_COMIC_SANS) || issmall(M))
-		update_flags |= M.adjustBruteLoss(-1, FALSE)
-		update_flags |= M.adjustFireLoss(-1, FALSE)
+		update_flags |= M.adjustBruteLoss(-1, FALSE, robotic = is_robot)
```

Comment:
```suggestion
		update_flags |= M.adjustBruteLoss(-1, FALSE, robotic = TRUE)
```
`is_robot` is a check on omnizine to check if it's the liquid metal variant or not. These should all just be set to `TRUE`.